core/src/main/java/org/eclipse/hono/util/MessageHelper.java

```diff
@@ -105,6 +105,10 @@
      * reported the data belongs to.
      */
     public static final String APP_PROPERTY_TENANT_ID = "tenant_id";
+    /**
+     * The name of the AMQP 1.0 message application property containing the resource version expected by the client.
+     */
+    public static final String APP_PROPERTY_RESOURCE_VERSION = "resource-version";
```

Comment:
please replace `-` with `_` so that it also works with JMS

Reply:
done